catalog/core/catalog-core-backupplugin/pom.xml

```diff
@@ -27,6 +29,22 @@
             <groupId>ddf.catalog.core</groupId>
             <artifactId>catalog-core-api-impl</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.hamcrest</groupId>
+            <artifactId>hamcrest-all</artifactId>
+            <version>1.3</version>
```

Comment:
Worth it / possible to update this version in the main ddf pom and then pull it in here?
```
<groupId>org.hamcrest</groupId>
<artifactId>hamcrest-all</artifactId>
<version>1.3</version>
```